component/Navigation/index.js

```diff
@@ -0,0 +1,24 @@
+import Link from "next/link";
+import {
+  StyledLink,
+  StyledNavLinkAchor,
+  StyledNavLinkText,
+  StyledNavList,
+} from "./Navigation.styled";
+
+export default function Navigation() {
+  return (
+    <nav>
+      <StyledNavList>
+        <StyledLink>
+          <StyledNavLinkAchor href="/">🏠</StyledNavLinkAchor>
+          <StyledNavLinkText href="/">Home</StyledNavLinkText>
+        </StyledLink>
```

Comment:
It is rather odd that you have two links right next to each other directing to the same page. And having a styled component that is called StyledLink while being a list item, is not ideal as well.

I would recommend having a link wrapping your icon and your text. 
Since you want your icon to have a font-size of 2em, I recommend wrapping it in something like a span and assigning the font-size property.

In the end you will have an unordered list wrapping your list items. Those list items contain only one link to the respective pages.